### ISSUE TITLE
Unable to build test aepp for production, found circular deps in SDK

### ISSUE THREAD

**Issue by quantum0hound** [aeternity/aepp-sdk-js#1421]
When I serve an aepp to production, I receive such error in browser as: 
```
schema.js:475 Uncaught ReferenceError: Cannot access 'dr' before initialization
    at Object.52373 (schema.js:475:20)
    at n (bootstrap:19:32)
    at startup:4:87
    at Function.n.O (chunk loaded:25:12)
    at startup:5:43
    at app.2e26927d.js:1:114649
```
I've found that error is caused by WebPack bundling and found [https://github.com/webpack/webpack/issues/12724].
Then I've added circular-dependency-plugin to vue.config.js and received:
```
 ERROR  Failed to compile with 6 errors                                                                                                                                                1:57:44 PM

 error  

Circular dependency detected:
src/@aeternity/aepp-sdk/es/channel/handlers.mjs -> src/@aeternity/aepp-sdk/es/channel/internal.mjs -> src/@aeternity/aepp-sdk/es/channel/handlers.mjs

 error  

Circular dependency detected:
src/@aeternity/aepp-sdk/es/channel/internal.mjs -> src/@aeternity/aepp-sdk/es/channel/handlers.mjs -> src/@aeternity/aepp-sdk/es/channel/internal.mjs

 error  

Circular dependency detected:
src/@aeternity/aepp-sdk/es/tx/builder/field-types.mjs -> src/@aeternity/aepp-sdk/es/tx/builder/helpers.mjs -> src/@aeternity/aepp-sdk/es/tx/builder/schema.mjs -> src/@aeternity/aepp-sdk/es/tx/builder/field-types.mjs

 error  

Circular dependency detected:
src/@aeternity/aepp-sdk/es/tx/builder/helpers.mjs -> src/@aeternity/aepp-sdk/es/utils/crypto.mjs -> src/@aeternity/aepp-sdk/es/tx/builder/helpers.mjs

 error  

Circular dependency detected:
src/@aeternity/aepp-sdk/es/tx/builder/schema.mjs -> src/@aeternity/aepp-sdk/es/tx/builder/field-types.mjs -> src/@aeternity/aepp-sdk/es/tx/builder/helpers.mjs -> src/@aeternity/aepp-sdk/es/tx/builder/schema.mjs

 error  

Circular dependency detected:
src/@aeternity/aepp-sdk/es/utils/crypto.mjs -> src/@aeternity/aepp-sdk/es/tx/builder/helpers.mjs -> src/@aeternity/aepp-sdk/es/utils/crypto.mjs
```

What can be done to avoid such errors?

**Comment by marc0olo** [comment]
not sure about the problem here. can you share the project / sourcecode? which version of SDK are you using?

when you speak about Vue, did you check out the Vue Boilerplate? it was recently updated:
- https://github.com/aeternity/aepp-boilerplate-vue

**Comment by quantum0hound** [comment]
https://github.com/quantum0hound/AeTokenTransfer , using latest SDK "github:aeternity/aepp-sdk-js#develop". Will check the boilerplate, thanks

**Comment by marc0olo** [comment]
we've generally introduced lot of major and breaking changes recently that produced some unforeseen problems in `v10.0.0`. it should all be fixed with the upcoming `v11.0.0`.

please don't be too frustrated if sth. doesn't work as expected. we're happy to help out!

**Comment by quantum0hound** [comment]
Boilerplare works just fine, thanks for rapid support
